src/site/apt/filemappers.apt

```diff
@@ -94,3 +98,22 @@ File Mappers
 -----------------------------------------------------------------------------
 
   The merging file mapper uses the role hint "merge".
+  
+* {Suffix File Mapper}
+
+  The {{{./apidocs/org/codehaus/plexus/components/io/filemappers/SuffixFileMapper.html}suffix
+  file mapper}} add the given suffix to the filename. The suffix will be added before the file
+  extension. Examples : 
+  theFile.txt => theFileNiceSuffix.txt
```

Comment:
The examples will be displayed on one line as the new lines will be stripped. If you wrap them as code that would solve the problem and they would stand out as well. And would be great if you add example with dot in the file name as it would make clear how the mapper behaves in such cases.

Reply:
I have still the following example : dir/archive.tar.gz => dir/archiveNiceSuffix.tar.gz
this is not enough ? If not please tell me an example

Reply:
It is ok. My bad, didn't read the docs carefully enough. Sorry about that.